tests/test_geoarrow.py

```diff
@@ -109,3 +109,55 @@ def test_from_geoarrow_invalid_encoding():
 def test_from_geoarrow_no_arrow_object():
     with pytest.raises(ValueError, match="input should be an Arrow-compatible array"):
         spherely.from_geoarrow(np.array(["POINT (1 1)"], dtype=object))
+
+
+def test_to_geoarrow():
+    arr = spherely.points([1, 2, 3], [1, 2, 3])
+    res = spherely.to_geoarrow(
+        arr, output_schema=ga.point().with_coord_type(ga.CoordType.INTERLEAVED)
+    )
+    assert isinstance(res, spherely.ArrowArrayHolder)
+    assert hasattr(res, "__arrow_c_array__")
+
+    arr_pa = pa.array(res)
+    coords = np.asarray(arr_pa.storage.values)
+    expected = np.array([1, 1, 2, 2, 3, 3], dtype="float64")
+    np.testing.assert_allclose(coords, expected)
+
+
+def test_to_geoarrow_wkt():
+    arr = spherely.points([1, 2, 3], [1, 2, 3])
+    result = pa.array(spherely.to_geoarrow(arr, output_schema=ga.wkt()))
+    expected = pa.array(["POINT (1 1)", "POINT (2 2)", "POINT (3 3)"])
+    assert result.storage.equals(expected)
+
+
+def test_to_geoarrow_wkb():
+    arr = spherely.points([1, 2, 3], [1, 2, 3])
+    result = pa.array(spherely.to_geoarrow(arr, output_schema=ga.wkb()))
+    expected = ga.as_wkb(
+        [
+            "POINT (0.9999999999999998 1)",
+            "POINT (2 1.9999999999999996)",
+            "POINT (3.0000000000000004 3.0000000000000004)",
```

Comment:
This looks a bit weird?

Reply:
Yes, this looks weird, but I don't know a better way to test it ..

The general problem with those test assertions is that we test in lon/lat space, but s2geometry converts the input lon/lat to unit vector, and then converting back to lon/lat (to WKB in this case) will not give the exact same values as the input values. 

For testing with WKT, this is generally fine because we limit the precision of the output, and then you don't see those small differences. But for the WKB output, as a binary value, that contains the full exact float values, so the above is needed if we want to verify the binary array result.

Of course, I could also then convert the WKB again to WKT and just verify that, but that is an additional step while I wanted to test the WKB result more directly (I am of course still creating the expected WKB from WKT, so maybe I could then just as well convert the result WKB to WKT and just verify that with the expected WKT ...)

Reply:
Thanks for the explanation! To me that's fine leaving it like this (maybe with a small comment summarizing what you explain here above).

Reply:
OK, added a comment!